flexmeasures/data/services/sensors.py

```diff
@@ -7,14 +7,25 @@
 
 
 def get_sensors(
+    account_id: int | None = None,
```

Comment:
I believe we should keep this to one argument. 

If it's int: query for account ID.
If it's string: query for name.
Otherwise, it's an error.

Reply:
Thanks. Your idea also gave me the idea to avoid some duplicate queries to the Account table, and to filter the query of the sensors table.